tutorial-3.1/common_files/nacl_prod.py

```diff
@@ -1,6 +1,6 @@
-from openmm.app import *
-from openmm import *
-from openmm.unit import *
+from simtk.openmm.app import *
```

Comment:
Using simtk.openmm imports is deprecated. 

https://github.com/openmm/openmm/releases/tag/7.6.0

Reply:
I tried it without that and it didn't work for me... do you know why that might be the case?

Reply:
Ah using version 7.7 worked, I must have been on 7.5 when I initially tried it.  I'll fix those imports.